.github/workflows/humble-pre-release.yml

```diff
@@ -0,0 +1,27 @@
+name: Humble - pre-release
+# author: Christoph Froehlich <christoph.froehlich@ait.ac.at>
+
+on:
+  workflow_dispatch:
+    inputs:
+      downstream_depth:
+        description: 'The depth of the depends-on tree to be included in the overlay workspace (-1 implies unlimited, default: 0)'
+        required: false
+        default: 0
+        type: number
+  pull_request:
+    branches:
+      - humble
+    types:
+      - opened # default
+      - reopened # default
+      - synchronize # default
+      - labeled # also if a label changes
+
+jobs:
+  default:
+    uses: ros-controls/ros2_control_ci/.github/workflows/reusable-prerelease.yml@reusable_prerelease
+    with:
+      ros_distro: humble
+      # downstream_depth is not set on pull_request event
+      prerelease_downstream_depth: ${{ github.event_name == 'pull_request' && '0' || inputs.downstream_depth }}
```

Comment:
Do we need the '0' in the condition?

Reply:
This is kind of ternary if operator, the only way in doing something like this in github workflow syntax.
`${{ github.event_name == 'pull_request' || inputs.downstream_depth }}`
will result in 
`PRERELEASE_DOWNSTREAM_DEPTH: true`
for PRs and
`generate_prerelease_script.py: error: argument --level: invalid int value: 'true'`